projects/plugins/jetpack/modules/search/customberg/components/sidebar/index.jsx

```diff
@@ -1,113 +1,47 @@
- */
-function Tabs( { currentTab } ) {
-	return (
-		<ul>
-			<li>
-				<Tab
-					identifier={ JP_SEARCH_TAB_IDENTIFIER }
-					label={ __( 'Jetpack Search', 'jetpack' ) }
-					isActive={ currentTab === JP_SEARCH_TAB_IDENTIFIER }
-				/>
-			</li>
-			<li>
-				<Tab
-					identifier={ OPTIONS_TAB_IDENTIFIER }
-					label={ __( 'Options', 'jetpack' ) }
-					isActive={ currentTab === OPTIONS_TAB_IDENTIFIER }
-				/>
-			</li>
-		</ul>
-	);
-}
-
-/**
- * Sidebar tab.
- *
- * @param {object} props - Component properties
- * @param {string} props.identifier - Identifier.
- * @param {string} props.label - Label.
- * @param {boolean} props.isActive - Whether the tab is active.
- * @returns {Element} component instance
- */
-function Tab( { identifier, label, isActive } ) {
-	const { enableComplementaryArea } = useDispatch( interfaceStore );
-	// translators: %s: sidebar label e.g: "Options".
-	const ariaLabel = isActive ? sprintf( __( '%s (selected)', 'jetpack' ), label ) : label;
-	return (
-		<Button
-			// eslint-disable-next-line react/jsx-no-bind
-			onClick={ () => enableComplementaryArea( COMPLEMENTARY_AREA_SCOPE, identifier ) }
-			className={ classNames( 'jp-search-configure-sidebar__panel-tab', {
-				'is-active': isActive,
-			} ) }
-			aria-label={ ariaLabel }
-			data-label={ label }
-		>
-			{ label }
-		</Button>
-	);
-}
-
 /**
  * Sidebar implemented via ComplementaryArea component. Renders using the slot/fill paradigm.
  *
  * @returns {Element} component instance
  */
-export default function Sidebar() {
-	const currentTab = useSelect( select => {
-		const { getActiveComplementaryArea } = select( interfaceStore );
-		return getActiveComplementaryArea( COMPLEMENTARY_AREA_SCOPE )
-			? getActiveComplementaryArea( COMPLEMENTARY_AREA_SCOPE )
-			: OPTIONS_TAB_IDENTIFIER;
-	}, [] );
+export default function Sidebar( props ) {
+	const { enabledSidebarName, enableSidebar, disableSidebar } = props;
 
 	return (
-		<ComplementaryArea
-			className="jp-search-configure-sidebar"
-			closeLabel={ __( 'Close settings', 'jetpack' ) }
-			header={ <Tabs currentTab={ currentTab } /> }
-			headerClassName="jp-search-configure-sidebar__panel-tabs"
-			icon={ cog }
-			identifier={ currentTab }
-			isActiveByDefault
-			scope={ COMPLEMENTARY_AREA_SCOPE }
-			/* translators: button label text should, if possible, be under 16 characters. */
-			title={ __( 'Settings', 'jetpack' ) }
-		>
-			{ currentTab === JP_SEARCH_TAB_IDENTIFIER && <SidebarDescription /> }
-			{ currentTab === OPTIONS_TAB_IDENTIFIER && <SidebarOptions /> }
-		</ComplementaryArea>
+		<div className="interface-complementary-area jp-search-configure-sidebar">
+			<div
+				className="components-panel__header interface-complementary-area-header jp-search-configure-sidebar__panel-tabs"
+				tabindex="-1"
+			>
+				<Tabs enabledSidebarName={ enabledSidebarName } enableSidebar={ enableSidebar } />
+				<Button
+					ariaLabel={ __( 'Close settings', 'jetpack' ) }
```

Comment:
should `ariaLabel` here be `aria-label`?

Reply:
Yep - oddly these aren't camel-cased in React:

https://reactjs.org/docs/accessibility.html

Reply:
👍